GVRf/Framework/jni/objects/mesh.cpp

```diff
@@ -145,6 +146,66 @@ const float* Mesh::getBoundingBoxInfo() {
     return bounding_box_info_;
 }
 
+float* Mesh::getTransformedBoundingBoxInfo(glm::mat4 *Mat) {
+
+    if (have_bounding_box_ == false)
+        getBoundingBoxInfo();
+
+    glm::mat4 M = *Mat;
+    float a, b;
+    float *transformed_bounding_box = new float[6];
+
```

Comment:
Just minor concern here. Probably you can create a member float array of size 6 instead of doing new and delete the array each call.